### ISSUE TITLE
Image has a long description attribute

### ISSUE THREAD

**Issue by alvaromontoro** [jackdomleo7/Checka11y.css#108]
### Describe the new a11y feature or project enhancement

The [`longdesc`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#attr-longdesc) attribute in images is deprecated and should not be used. It is not fully supported by HTML5, browsers, or screen readers. When present, it is commonly misused (it should point to an URL that is not an image).

### Describe the solution you'd like

Show a warning if:

- an image has the `longdesc` attribute. Recommend the removal of the attribute and adding the information on the same page (e.g. with an `aria-describedby`)

Show an error if:

- The value of `longdesc` is not a URL (this may be tricky as it could be a relative path). 
- The value of `longdesc` contains spaces (then it would be text and not a URL)
- The value of `londgesc` is a link to an image. This could be tested via CSS by checking if the end of the value is an image extension (e.g. ".jpg", ".jpeg", ".png", ".gif", etc.)
- The value of `longdesc` is empty.

Testing the first two points may be tricky and lead to false positives (e.g. a URL that has not been encoded and contains white space). Because of that, maybe those two cases should be ignored.

### Link(s)

- [Equalize Digital: Long Description Invalid](https://equalizedigital.com/accessibility-checker/long-description-invalid/)
- [WebAIM: The WebAIM Million (longdesc)](https://webaim.org/projects/million/#longdesc)

**Comment by jackdomleo7** [comment]
This is a brilliant idea and a nice feature to add to a future version 👌

**Comment by Deep1Shikha** [comment]
@jackdomleo7 , I would like to pick this issue. Please assign it to me.

**Comment by jackdomleo7** [comment]
@Deep1Shikha absolutely 🙂

**Comment by jackdomleo7** [comment]
Accidentally clicked the close button - _for auditing_

**Comment by alvaromontoro** [comment]
@Deep1Shikha are you still working on this one? If not, maybe I can pick it up :)

**Comment by jackdomleo7** [comment]
Doesn't look like @Deep1Shikha has made any progress yet (https://github.com/Deep1Shikha/Checka11y.css). Maybe we could give them a week to respond and if not, I can assign to @alvaromontoro?

**Comment by alvaromontoro** [comment]
@jackdomleo7 assign it to me. I will work on it this week(end).
